docs/notebooks/rembo.pct.py

```diff
@@ -0,0 +1,246 @@
+# %% [markdown]
+# # Noise-free optimization on a toy high-dimensional problem using Random EMbedding Bayesian Optimization (REMBO).
+# This notebook demonstrates a simple method for optimizing a high-dimensional (100-D) problem, where standard BO methods have trouble.
+
+# %%
+
+import math
+import gpflow
+import numpy as np
+import tensorflow as tf
+import tensorflow_probability as tfp
+
+np.random.seed(1793)
+tf.random.set_seed(1793)
+
+# %% [markdown]
+# ## Describe the problem
+#
+# In this example, we augment the standard two-dimensional Michalewicz function with 98 dummy dimensions to obtain a 100-dimensional problem over the hypercube $[0, \pi]^{100}$.
```

Comment:
why not Branin? something we know standard method would perform extremely well on?

Reply:
Branin didn't quite show the results we wanted as well, it's a bit too easy